Sources/StreamChatUI/ChatChannel/ChatChannelVC.swift

```diff
@@ -258,12 +260,14 @@ open class ChatChannelVC: _ViewController,
 
     /// Marks the channel read and updates the UI optimistically.
     public func markRead() {
-        channelController.markRead { [weak self] error in
-            if error == nil {
-                self?.hasSeenLastMessage = false
+        throttler.throttle { [weak self] in
```

Comment:
Nit: This will make it more consistent with our `Debouncer`
```suggestion
        throttler.execute { [weak self] in
```

Reply:
We will handle this later 👍